UnitsNet/CustomCode/Quantities/Energy.extra.cs

```diff
@@ -19,6 +19,12 @@ public partial struct Energy
             return Power.FromWatts(energy.Joules / duration.Seconds);
         }
 
+        /// <summary>Get <see cref="TimeSpan"/> from <see cref="Energy"/> divided by <see cref="Power"/>.</summary>
+        public static TimeSpan operator /(Energy energy, Power power)
```

Comment:
Can we change TimeSpan to Duration?

This way it is an `IQuantity`. We usually support both Duration and TimeSpan in overloads that take duration as a parameter, but when it is the result we prefer Duration.

Reply:
Perfect, was wondering which one was preferred. Have updated 🙌